test/admin/e2e/list-view/e2e.spec.ts

```diff
@@ -608,6 +615,35 @@ describe('List View', () => {
     })
   })
 
+  describe('WhereBuilder', () => {
```

Comment:
There should already be a "filter options" describe block in this file. Can move this test to there.

Reply:
Yes, it's the one above. I thought so too. But I needed the collection to have more than 10 documents to test the scroll so I created another collection.

Reply:
You can still do this, just create more documents within your test. No need for another collection really.

Reply:
Yes, I know. But I don't know if I break another test until I run the entire suite, which has already happened to me. I'm going to leave it as is if you don't mind.